client/app/components/onboardingGuide/GuideSloganAndDescriptionPage.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { PropTypes } from 'react';
+import scss from './styles.scss';
+
+const GuideSloganAndDescriptionPage = (props) => {
+  const handleClick = function handleClick(e, path) {
+    e.preventDefault();
+    props.changePage(path);
+  };
+
+  return (
+    <div className="container">
+      <a className={scss.backLink}
+        onClick={(e) => handleClick(e, '')}
+        href={props.initialPath}
+      >
+        ‹ {props.t('back_to_todo')}
+      </a>
```

Comment:
As this link is in many components and in most cases `handleClick` is mostly used here, making it it's own component would be a good idea. Maybe even before merging, can't say!